inflate.c

```diff
@@ -1332,13 +1333,11 @@ int32_t Z_EXPORT PREFIX(inflateCopy)(PREFIX3(stream) *dest, PREFIX3(stream) *sou
     if (copy == NULL)
         return Z_MEM_ERROR;
     window = NULL;
```

Comment:
You need to remove `window` reference. And use `copy->window` below.

Reply:
Thanks, completely missed that, as did the `infcover` test..
It fails with a segfault now however, but why.. Time to have a look with gdb I guess.

Reply:
`state->wbits` and `state->wsize` were the culprits, had to reorganize the code somewhat since ensure_window does not have access to the source state, and only had the new blank state.

This passes tests and valgrind now.